magento/pwa-studio
# Mini Cart V2 / Shopping Bag Footer section.

## Description

Added footer section for the new Mini Cart / Shopping Bag component.

**Should be merged only after #2509** 

## Related Issue
Closes [PWA-601](https://jira.corp.magento.com/browse/PWA-601)

### Verification Stakeholders
@jimbo 
@soumya-ashok 
@schensley 

### Verification Steps

1. run `yarn watch:venia`.
2. Add an item to the cart and click the new cart button.
3. We should be able to see the footer section.

## Screenshots / Screen Captures (if appropriate)

![image](https://user-images.githubusercontent.com/35203638/85621071-5211c780-b62a-11ea-8afb-00c835f4a7c3.png)

## Checklist

* **Should be merged only after #2509**
